src/alexapi/player.py

```diff
@@ -0,0 +1,302 @@
+import vlc
+import time
+import threading
+from collections import deque
+
+from shared import *
+import player_state as pstate
+
+player = None
+alexa_playback_progress_report_request = None
+alexa_getnextitem = None
+tuneinplaylist = None
+
+avr_playing = False
+media_playing = False
+media_paused = False
+
+class AlexaVoiceResponsePlayer(object):
+	instance = None
```

Comment:
Call this _vlc_, _vlcInstance_ or something. Just _instance_ sounds like it's a static class variable that holds the instance in case of a singleton design pattern.